moped-editor/src/views/projects/ProjectSummary.js

```diff
@@ -19,234 +19,187 @@ import {
   Tabs,
   Typography,
   Grid,
+  CardActions,
 } from "@material-ui/core";
+import {
+  // rest of the elements/components imported remain same
+  useParams,
+} from "react-router-dom";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+  },
+  cardWrapper: {
+    marginTop: theme.spacing(3),
+  },
+  cardActions: {
+    display: "flex",
+    justifyContent: "flex-end",
+  },
+  button: {
+    marginTop: theme.spacing(1),
+    marginRight: theme.spacing(2),
+  },
+}));
+
+const SUMMARY_QUERY = gql`
+  query ProjectSummary($projectId: Int) {
+    moped_project(where: { project_id: { _eq: $projectId } }) {
+      project_name
+      project_description
+      start_date
+      current_phase
+      current_status
+      eCapris_id
+      fiscal_year
+      project_priority
+    }
+  }
+`;
+
+const TEAM_QUERY = gql`
+  query TeamSummary {
+    moped_proj_personnel(limit: 2, order_by: { project_personnel_id: desc }) {
+      first_name
+      last_name
+      role_name
+      notes
+    }
+  }
+`;
 
-const ProjectSummary = () => {
-  const useStyles = makeStyles(theme => ({
-    root: {
-      flexGrow: 1,
-      backgroundColor: theme.palette.background.paper,
-    },
-    cardWrapper: {
-      marginTop: theme.spacing(3),
-    },
-    buttons: {
-      display: "flex",
-      justifyContent: "flex-end",
-    },
-    button: {
-      marginTop: theme.spacing(3),
-      marginLeft: theme.spacing(1),
-    },
-  }));
-  const classes = useStyles();
-
-  const navigate = useNavigate();
-
-  const handleClick = () => {
-    navigate("/moped/projects");
-  };
+function TabPanel(props) {
```

Comment:
It never occurred to me to have two components within a file, this is clever, but maybe split into two different files?

Reply:
yeah, I definitely agree that this Project Summary component needs to be broken down into smaller components. I started doing a bit of that, but for the sake of time I just keep pushing ahead. This could definitely use more refactoring.